common/util.go

```diff
@@ -313,7 +313,7 @@ func CreateMatchingPollForDecisionTaskResponse(historyResponse *types.RecordDeci
 	matchingResp := &types.MatchingPollForDecisionTaskResponse{
 		WorkflowExecution:         workflowExecution,
 		TaskToken:                 token,
-		Attempt:                   Int64Ptr(historyResponse.GetAttempt()),
+		Attempt:                   historyResponse.GetAttempt(),
```

Comment:
huh. what type is this actually?  can it just be `historyResponse.Attempt`?
but generally: `Get` -> field can wait if you want, we can tackle those later.

Reply:
It can and it should be `historyResponse.Attempt`. All those helpers will become useless now, and should be simplified. However I'm planning to do it separately.